pkg/kubernetes/pod/v1alpha1/build.go

```diff
@@ -40,6 +40,19 @@ func NewBuilder() *Builder {
 	return &Builder{pod: &Pod{object: &corev1.Pod{}}}
 }
 
+// WithTolerations sets the Spec.Tolerations withh provided value.
+func (b *Builder) WithTolerations(tolerations []corev1.Toleration) *Builder {
```

Comment:
toleration can be more then 1 , so having func `WithTolerations(tolerations []corev1.Toleration...)` is a better

Reply:
```go
func (b *Builder) WithTolerationsByValue(tolerations ...corev1.Toleration) *Builder {
	if len(b.pod.object.Spec.Tolerations) == 0 {
		b.pod.object.Spec.Tolerations = tolerations
		return b
	}

	b.pod.object.Spec.Tolerations = append(
		b.pod.object.Spec.Tolerations,
		tolerations...,
	)

	return b
}
```

Reply:
As suggested by @prateekpandey14  variadic parameters will be better than giving a slice of tolerations.